fixtures/_fixtures/logger.py

```diff
@@ -99,7 +102,11 @@ def setUp(self):
         self._output = output
         handler = StreamHandler(output)
         if self._format:
-            handler.setFormatter(Formatter(self._format, self._datefmt))
+            if self._formatter:
+                handler.setFormatter(
+                    self._formatter(self._format, self._datefmt))
+            else:
+                handler.setFormatter(Formatter(self._format, self._datefmt))
```

Comment:
If I read this correctly, it is equivalent to

 formatter = (self._formatter or Formatter)(self._format, self._datefmt)
 handler.setFormatter(formatter)

Which I think is clearer.

The patch looks fine, but please do add a unittest.